Sources/PrivoSDK/internal/PrivoCameraPermissionService.swift

```diff
@@ -1,7 +1,14 @@
 import Foundation
```

Comment:
how about to make a global PrivoPermission service, where camera is only first case in it ?